e2e/bin/nodes-opts

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+nodes=$(terraform output -json -state=terraform/terraform.tfstate | jq -r '(.linux_clients,.servers).value[]')
+
+h=terraform/known_hosts
+if [ ! -f $h ]; then
+    ssh-keyscan $nodes > $h || exit 2
+fi
+
+opts=`echo -i terraform/keys/*.pem`
```

Comment:
The glob doesn't expand if you'd done `opts="-i terraform/keys/*.pem"`, does it?